server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -108,6 +108,10 @@ export async function runWithRetry<T>(
 		}
 	}
 
+	if (shouldIgnoreError !== undefined && shouldIgnoreError(latestResultError) === true) {
```

Comment:
Nit, can we do `shouldIgnoreError?(latestResultError) === true`?

Reply:
I updated to return from while block itself. Keeping the if condition as it is.